### PR TITLE
Updating CA guidance for VS2017

### PR DESCRIPTION
**Pull request by rub8n**
Adds a warning box with new guidance for using resources under CA policies

### CODE REVIEW

**Comment by PRMerger20** [comment]
@rub8n : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
@ornellaalt 

Could you review the proposed changes to your article?

Thanks!

File: docs/ide/signing-in-to-visual-studio.md [hunk 1]
```diff
@@ -18,6 +18,11 @@ You can personalize and optimize your development experience in Visual Studio by
 > [!NOTE]
 > This topic applies to Visual Studio on Windows. For Visual Studio for Mac, see [Sign in to Visual Studio for Mac](/visualstudio/mac/signing-in).
 
+> [!WARNING]
```
Comment (ornellaalt) [inline docs/ide/signing-in-to-visual-studio.md:21]: In order to limit this information to the page pertaining to VS2017, we need to use the Markdown moniker function. 
Replace all changes in this PR with:

::: moniker range="vs-2017"
> [! WARNING]
> Using Visual Studio 2017 to access resources configured for conditional access may trigger a degraded authentication experience, prompting reauthentication several times within the same Visual Studio session. 
> To work with resources configured for conditional access, upgrade to Visual Studio 2019 Update 16.6 or later. For more information, see [How to use Visual Studio with accounts that require multi-factor authentication](ide/work-with-multi-factor-authentication.md).

::: moniker-end

**Comment by ornellaalt** [comment]
#sign-off

**Comment by PRMerger20** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**